src/community/who-uses-dart.md

```diff
@@ -4,23 +4,88 @@ description: Mobile apps and Google web apps are two of the most common uses for
 toc: false
 ---
 
-The popular [Flutter toolkit][Flutter] relies on the Dart language,
-so developers writing Flutter apps—for Android, iOS, or other targets—use Dart.
-To find a list of Flutter apps, visit the [Flutter showcase.][showcase]
+## Flutter
 
-[Flutter]: {{site.flutter}}
-[showcase]: {{site.flutter}}/showcase
+The popular [Flutter toolkit][] is powered by the Dart language,
+runtimes, tooling, and libraries. Over 500,000 Flutter apps for Android,
+iOS, the web, and other targets, are all Dart-based.
 
-Google engineers use Dart to create many apps,
-including some that are essential to Google's business.
-For example, if you use the [Google Ads][] web or mobile app,
-you're using a Dart app that supports much of Google's revenue.
-Also, the Assistant team at Google uses Dart for features in Smart Displays,
-as described in [this announcement.][flutter-io19]
+The [Flutter showcase][] features popular Flutter apps, including:
 
+* [BMW][]
+    > "The BMW Group is establishing itself as a leading automotive app developer
```

Comment:
If we're keeping quotes, I think they should be ones that talk about Dart somehow.

Alternatively, we could just change this to short descriptions, and pull them up a level, e.g.:
```
* [BMW][]: A total of 300 developers use Dart to deliver a unified *My BMW app* for iOS and Android.
```